### DISCUSSION TITLE
Add permission for adding Dashboard widgets

### DISCUSSION THREAD

**Discussion by steveooo9** [craftcms/cms#8492] [Ideas]
### Description
Currently you aren't able to restrict adding and selecting Dashboard widgets for a user role. It would be great if you can disable the use of Dashboard widgets because sometimes the default widgets or widgets by 3rd-party plugins disclose some data nobody in this role should see.

**Comment by MoritzLost** [comment]
Any update on this? Would be great to have a single permission `Manage dashboard widgets` or something like this that just prevents user from editing the dashboard widgets. I'm working on a site with public registration and for those users, I want to provide default widgets that I don't want them to be able to change. Currently, the only way I've found is to hide the buttons with CSS and reset the widgets during login, which of course isn't foolproof:

```css
#newwidgetmenubtn, #widgetManagerBtn {
    display: none;
}
```

Pair that with the [Control Panel Body Classes](https://plugins.craftcms.com/cp-body-classes) plugin to apply those styles only to specific user groups. But having that built into the core as a permission that is enforced on the backend.
